examples/ip64-router/ip64-router.c

```diff
@@ -0,0 +1,27 @@
+#include "contiki.h"
+#include "contiki-net.h"
+#include "ip64.h"
```

Comment:
Oops I forgot to throw in the module, I'm testing for the Z1 and Re-Mote platforms.